src/app/project/ontology/resource-class-form/resource-class-form.component.ts

```diff
@@ -344,9 +367,58 @@ export class ResourceClassFormComponent implements OnInit, OnDestroy, AfterViewC
     submitData() {
         this.loading = true;
         if (this.edit) {
-            // edit mode
-            // submit properties and set cardinality
-            this.submitProps(this.resourceClassForm.value.properties, this.iri);
+
+            if (this.showResourceClassForm) {
+                // edit mode: res class info (label and comment)
+                // label
+                const onto4Label = new UpdateOntology<UpdateResourceClassLabel>();
+                onto4Label.id = this.ontology.id;
+                onto4Label.lastModificationDate = this.lastModificationDate;
+
+                const updateLabel = new UpdateResourceClassLabel();
+                updateLabel.id = this.iri;
+                updateLabel.labels = this.resourceClassLabels;
+                onto4Label.entity = updateLabel;
+
+                // comment
+                const onto4Comment = new UpdateOntology<UpdateResourceClassComment>();
+                onto4Comment.id = this.ontology.id;
+
+                const updateComment = new UpdateResourceClassComment();
+                updateComment.id = this.iri;
+                updateComment.comments = this.resourceClassComments;
+                onto4Comment.entity = updateComment;
+
+                this._dspApiConnection.v2.onto.updateResourceClass(onto4Label).subscribe(
+                    (classResponse: ResourceClassDefinitionWithAllLanguages) => {
+                        this.lastModificationDate = classResponse.lastModificationDate;
+                        onto4Comment.lastModificationDate = this.lastModificationDate;
+
+                        this._dspApiConnection.v2.onto.updateResourceClass(onto4Comment).subscribe(
+                            (classResponse: ResourceClassDefinitionWithAllLanguages) => {
```

Comment:
classResponse is shadowed here but it's not a big issue

Reply:
resolved in 076e0a7